src/utils/sortFavorably.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Sort favored items in the beginning of the list in given order.
+ * Then sort the rest of the items alphabetically.
+ */
+export default function sortFavorably(sortable: string[], favored: string[]) {
+  favored = [...favored].reverse();
```

Comment:
btw is it necessary to reverse  this in the first place, could handle ifs differently or is there some other idea behind this?

Reply:
without reverse "the rest of the languages" would be in the beginning and favored would be in the end of the returned array. I don't know why, but it just is so :D 🤯